tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -222,9 +202,14 @@
         // Which tab is selected (scalars, graph, images etc).
```

Comment:
If this comment is accurate, can it be updated to reference "plugin name", assuming that the tab name is always the plugin name.